programs/zstdcli.c

```diff
@@ -196,6 +196,7 @@ static void usage_advanced(const char* programName)
     DISPLAYOUT( " -l     : print information about zstd compressed files \n");
     DISPLAYOUT( "--test  : test compressed file integrity \n");
     DISPLAYOUT( " -M#    : Set a memory usage limit for decompression \n");
+    DISPLAYOUT( "--no-check : disable validation of checksums in compressed frame \n");
```

Comment:
If both compression and decompression are compiled into the CLI, this will produce two help lines for `--no-check`, won't it?

We should consolidate them into one description line, which should vary based on whether both, neither, or one of compression and decompression are compiled in.

You might also, like the compression-side `--[no-]check`, want to describe and handle `--check`. (`zstd --no-check ... --check ...` should check, `zstd --check ... --no-check ...` shouldn't check. Is that already the case?)